v2_jobs_test.go

```diff
@@ -23,8 +23,13 @@ func TestJobs_List(t *testing.T) {
 		"link_uuid": "6ce08b95-1b3b-498a-8baa-e595e2ae7466",
 		"tasks": [
 			{
-				"uuid": "491aebbd-457b-4a6e-adf6-87a3a9ee951a",
-				"exit_code": 1
+				"uuid": "491aebbd-457b-4a6e-adf6-87a3a9ee951a", 
+				"exit_code": 1,
+				"file_name": "Images-94ade01c-49ce-49e0-9cc3-805575c676d0",
+				"time_created": "2024-01-18T01:27:49",
+				"time_started": "2024-01-18T01:27:49",
+				"time_ended": "2024-01-18T01:27:49",
+				"duration": "< 1"
```

Comment:
We should not make up this payload, use an example returned by Archivematica. I don't think we should see a string here, should be an integer.

Reply:
This is copy pasted from archivematica

Reply:
The API returns "< 1"? Wow. Then I guess we'll need a custom decoder that knows how to read that.

Reply:
yeah i thought it was weird when i saw it but yeah it actually returns that :laughing: